### ISSUE TITLE
scrabble: panic: interface conversion: ast.Expr is *ast.MapType, not *ast.Ident

### ISSUE THREAD

**Issue by dysolution** [exercism/exalysis#89]
Solution: https://exercism.io/mentor/solutions/73334574b15947d0b61e941092fed615
```
go test:         OK
go test -race:   SKIPPED
golint:          OK
gofmt:           OK
go vet:          OK
golangci-lint:   OK
benchmarks:      SKIPPED
panic: interface conversion: ast.Expr is *ast.MapType, not *ast.Ident

goroutine 1 [running]:
github.com/tehsphinx/astrav.(*CompositeLit).ValueType(0xc0000a2b00, 0x136e41f, 0xc0000510f0)
        /Users/jordan.peterson/pkg/mod/github.com/tehsphinx/astrav@v0.1.0/valuetype.go:26 +0x197
github.com/tehsphinx/astrav.(*baseNode).FindMaps.func1(0x13c6c40, 0xc0000a2b00, 0x0)
        /Users/jordan.peterson/pkg/mod/github.com/tehsphinx/astrav@v0.1.0/node.go:340 +0x38
github.com/tehsphinx/astrav.(*baseNode).TreeNodes(0xc0000a2888, 0x1380038, 0x0, 0x0, 0x0)
        /Users/jordan.peterson/pkg/mod/github.com/tehsphinx/astrav@v0.1.0/node.go:477 +0xd3
github.com/tehsphinx/astrav.(*baseNode).TreeNodes(0xc0000a2508, 0x1380038, 0x0, 0x0, 0x0)
        /Users/jordan.peterson/pkg/mod/github.com/tehsphinx/astrav@v0.1.0/node.go:480 +0x158
github.com/tehsphinx/astrav.(*baseNode).TreeNodes(0xc0000a2208, 0x1380038, 0xc00024faa0, 0x13c7b40, 0xc0000a2200)
        /Users/jordan.peterson/pkg/mod/github.com/tehsphinx/astrav@v0.1.0/node.go:480 +0x158
github.com/tehsphinx/astrav.(*baseNode).FindMaps(0xc0000a2208, 0x0, 0x0, 0x0)
        /Users/jordan.peterson/pkg/mod/github.com/tehsphinx/astrav@v0.1.0/node.go:339 +0x37
github.com/exercism/exalysis/track/scrabble.testMapInFunc(0xc0000be000, 0xc00031e0c0)
        /Users/jordan.peterson/src/github.com/exercism/exalysis/track/scrabble/scrabble.go:191 +0x61
github.com/exercism/exalysis/track/scrabble.Suggest(0xc0000be000, 0xc00031e0c0)
        /Users/jordan.peterson/src/github.com/exercism/exalysis/track/scrabble/scrabble.go:18 +0x7a
github.com/exercism/exalysis.GetSuggestions(0xc000016500, 0x42, 0xc0000bbea0, 0x1, 0x1, 0x55)
        /Users/jordan.peterson/src/github.com/exercism/exalysis/suggestion.go:66 +0x567
main.watchClipboard()
        /Users/jordan.peterson/src/github.com/exercism/exalysis/cmd/exalysis/main.go:64 +0x23e
main.main()
        /Users/jordan.peterson/src/github.com/exercism/exalysis/cmd/exalysis/main.go:31 +0x26a
```

**Comment by tehsphinx** [comment]
Yeah... had that one, too.

**Comment by tehsphinx** [comment]
Fixed in `v0.4.0`
